create_jenkins_job.py

```diff
@@ -287,23 +287,32 @@ def create_job(os_name, job_name, template_file, additional_dict):
                 'test_args_default': data['test_args_default'] + ' --packages-up-to rcpputils',
             })
 
-        # configure nightly job for compiling with clang+libcxx on linux
+        # configure job for compiling with clang+libcxx on linux
         if os_name == 'linux':
             # Set the logging implementation to noop because log4cxx will not link properly when using libcxx.
             clang_libcxx_build_args = data['build_args_default'].replace('--cmake-args',
                 '--cmake-args -DRCL_LOGGING_IMPLEMENTATION=rcl_logging_noop') + \
```

Comment:
While we are in here, we might consider removing forcing the logging implementation to 'noop'.  I know for sure that it works with spdlog.  I know it isn't strictly related to this PR, but I still think it is worth doing.